chart/charts/crds/templates/csi-volume-snapshot.yaml

```diff
@@ -7,6 +7,7 @@ metadata:
   annotations:
```

Comment:
Should we also remove the crdName check above? That was added post your change because upgrades would otherwise fail:
> Error: UPGRADE FAILED: Unable to continue with update: CustomResourceDefinition "volumesnapshotclasses.snapshot.storage.k8s.io" in namespace "" exists and cannot be imported into the current release: invalid ownership metadata; label validation error: missing key "app.kubernetes.io/managed-by": must be set to "Helm"; annotation validation error: missing key "meta.helm.sh/release-name": must be set to "mayastor"; annotation validation error: missing key "meta.helm.sh/release-namespace": must be set to "mayastor"

I think it was maybe not a good approach though, as it's perhaps better to simply document that for upgrades a user should disable these with --set crds.enabled=false?

Reply:
I also believe it's better to remove the `crdIsAbsent` check everywhere. I'll proceed with the change.

Reply:
addressed in c53b06999633e54b7e550bbfb9a4523fe763efb7